AWSCognitoAuth/AWSCognitoAuth.m

```diff
@@ -473,37 +560,78 @@ - (NSString *)getQueryStringSuffixForParameters:(NSDictionary<NSString *, NSStri
  */
 - (void) signOutInternal: (UIViewController *) vc completion: (AWSCognitoAuthSignOutBlock) completion {
     self.signOutBlock = completion;
-    NSString *url = [NSString stringWithFormat:@"%@?%@",
+    NSString *urlString = [NSString stringWithFormat:@"%@?%@",
                      self.authConfiguration.signOutUri,
                      [self getQueryStringSuffixForParameters:self.authConfiguration.signOutUriQueryParameters]];
+    NSURL *url = [NSURL URLWithString:urlString];
+    if(@available(iOS 13.0, *)) {
+        if (self.presentationAnchor) {
+            [self launchASWebAuthenticationSessionForSignOut: url];
+        } else {
+            [self launchLegacySessionForSignOut:url withPresentingViewController:vc];
+        }
+    } else {
+        [self launchLegacySessionForSignOut:url withPresentingViewController:vc];
+    }
+}
 
+
+- (void)launchSFAuthenticationSessionForSignOut:(NSURL *) url API_AVAILABLE(ios(11.0)) {
```

Comment:
Nit: See above about note on method ordering: signOutInternal calls `launchASWebAuthenticationSessionForSignOut` and `launchLegacySessionForSignOut`, which in turn calls `launchSFAuthenticationSessionForSignOut` and `signOutSFSafariVC`--I'd expect to see the methods in roughly that order

Reply:
Ordered as per the flow.